<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgorythmRun</class>
 <widget class="QMainWindow" name="AlgorythmRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>186</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>186</width>
      <height>30</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QToolButton" name="PrevButton">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>New</string>
     </property>
     <property name="text">
      <string>Prev</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="NextButton">
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>New</string>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="GraphField">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>96</y>
      <width>1231</width>
      <height>511</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="GraphFieldLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
